pages/linux/lnav.md

```diff
@@ -0,0 +1,16 @@
+# lnav
+
+> Advanced log file viewer to analyze logs with little to no setup.
+> More information: <https://lnav.org>.
+
+- View logs of a program, the arguments can be log files, directories or URLs:
+
+`lnav {{path/to/logdir}}`
```

Comment:
```suggestion
`lnav {{path/to/log_directory}}`
```

Reply:
Yeah, it looks more sensible now. Thanks for suggestion.